community/hacktoberfest/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: single
+title:  "Hacktoberfest"
+sidebar:
+  nav: community
+---
+
+[Hacktoberfest](https://hacktoberfest.digitalocean.com/) is a month-long global celebration of open source software that happens every year in October. During this event, anyone can support open source by contributing changes, which earn swag and recognition from the [Continuous Delivery Foundation](https://cd.foundation), our parent organization.
+
+We invite everyone to participate and to contribute to the Spinnaker project, regardless of their background and Spinnaker experience. There are many ways to contribute to Spinnaker during Hacktoberfest and other contributor events. You can work on code or documentation, assist community members in [Spinnaker Slack](http://join.spinnaker.io), or answer [StackOverflow questions](https://stackoverflow.com/questions/tagged/spinnaker). You can also share your experiences with Spinnaker by creating a new blog post, video, or social media steam. For more information on eligible contributions, see the [What to Hack](/community/gardening/what-to-hack/) page. For help getting started as a Spinnaker contributor, visit the [Contributing](/community/contributing/) page.
+
+All pull requests, issues, and PR or issue comments made to the `spinnaker` and `spinnaker-hackathon` organizations on GitHub count as contributions. Here are some ideas to get you started:
+- [Beginner-friendly issues on GitHub](https://github.com/spinnaker/spinnaker/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+friendly%22)
+- [Project ideas](https://github.com/spinnaker-hackathon/gardening/blob/master/project-ideas.md) from Gardening Days, the Spinnaker hackathon program
+
+### Hacktoberfest versus Gardening Days
+
+The Spinnaker community is experimenting with new ways to invite contributors into our community, and that may cause some confusion. We launched the Spinnaker Gardening Days hackathon program this year and have created new resources meant to assist contributors in those events and beyond. Thus, if you are participating in CDF's Hacktoberfest and explore Spinnaker contribution, you may see links to Gardening Days resources. Feel free to use those resources to help you get set up to contribute as part of Hacktoberfest. YOu can also use the [#gardening-general](https://spinnakerteam.slack.com/archives/CV4A90DPF) channel in [Slack](http://join.spinnaker.io) to ask questions about contributing and get to know the community.
```

Comment:
Next year, "this year" will now be false. 😁

Reply:
true, will get rid of it. I used that because I think this page will need to be nuked next year for a variety of reasons but...point taken